### PR TITLE
4.14 GA Bugs for Dev Console

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.14

Issue:
[Bug Batching for 4.14 GA](https://issues.redhat.com/browse/OSDOCS-5987)

Link to docs preview:
[Preview](https://66213--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bug-fixes)

Additional information:
[Dev Console Bugs](https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12347491#SIGwKWmOqDAaIiGh1DSCAiYg6oaN4RY8HF6AMAwlQMHEVU1ccdWpPFMCNc1SXIGgGCNeaPAtG0HT+ogCErGsyGEN+BWVIilTqkCILgh+5JftSCgaB47qFfMcT8UaPCIpgK00BoGjNXUCyFO6MRwntuwWkJVhAA)

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1305,6 +1305,18 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-dev-console-bug-fixes"]
 ==== Developer Console
 
+* Previously, if you tried to edit a Helm chart repository in the *Developer* console by navigating to *Helm*, clicking the *Repositories* tab, then selecting *Edit HelmChartRepository* through the kebab menu for your Helm chart repository, an *Error* page was displayed that showed a *404: Page Not Found* error. This was caused by a component path that was not up to date. This issue is now fixed. (link:https://issues.redhat.com/browse/OCPBUGS-14660[*OCPBUGS-14660*])
+
+* Previously, it was difficult to distinguish between the types of samples listed in the *Samples* page. With this fix, you can easily identify the sample type from the badges displayed on the *Samples* page. (link:https://issues.redhat.com/browse/OCPBUGS-7446[*OCPBUGS-7446*])
+
+* Previously, on the Pipeline *Metrics* page only four legends are visible for TaskRun duration charts. With this update, you can see all the legends present for the TaskRun duration charts. (link:https://issues.redhat.com/browse/OCPBUGS-19878[*OCPBUGS-19878*])
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1312]:
```suggestion
* Previously, on the Pipeline *Metrics* page, only four legends are visible for TaskRun duration charts. With this update, you can see all the legends present for the TaskRun duration charts. (link:https://issues.redhat.com/browse/OCPBUGS-19878[*OCPBUGS-19878*])
```
or maybe:
```suggestion
* Previously on the Pipeline *Metrics* page, only four legends are visible for TaskRun duration charts. With this update, you can see all the legends present for the TaskRun duration charts. (link:https://issues.redhat.com/browse/OCPBUGS-19878[*OCPBUGS-19878*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1305,6 +1305,18 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, an issue occurred when creating an application using the `Import JAR` form in a disconnected cluster with the Cluster Samples Operator not installed. With this update, the `Import JAR` form from the *Add* page and the *Topology* page. (link:https://issues.redhat.com/browse/OCPBUGS-15011[*OCPBUGS-15011*])
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1314]: Is the 2nd sentence missing something? Or maybe I'm reading it wrong.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -1305,6 +1305,18 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, the Operator backed catalog did not show any catalog items if CSV copies were disabled. With this fix, Operator backed catalogs are shown in every namespace even if CSV copies are disabled. (link:https://issues.redhat.com/browse/OCPBUGS-14907[*OCPBUGS-14907*])
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1316]:
```suggestion
* Previously, the Operator backed catalog did not show any catalog items if cluster service version (CSV) copies were disabled. With this fix, Operator backed catalogs are shown in every namespace even if CSV copies are disabled. (link:https://issues.redhat.com/browse/OCPBUGS-14907[*OCPBUGS-14907*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -1305,6 +1305,18 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, in the *Import from Git* and *Deploy Image* flows, the *Resource Type* section was moved to *Advanced* section. As a result, it was difficult to identify the type of resource created. With this fix, *Resource Type* section is moved to the *General* section. (link:https://issues.redhat.com/browse/OCPBUGS-7395[*OCPBUGS-7395*])
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1318]:
```suggestion
* Previously, in the *Import from Git* and *Deploy Image* flows, the *Resource Type* section was moved to *Advanced* section. As a result, it was difficult to identify the type of resource created. With this fix, the *Resource Type* section is moved to the *General* section. (link:https://issues.redhat.com/browse/OCPBUGS-7395[*OCPBUGS-7395*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -1690,6 +1690,18 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-dev-console-bug-fixes"]
 ==== Developer Console
 
+* Previously, if you tried to edit a Helm chart repository in the *Developer* console by navigating to *Helm*, clicking the *Repositories* tab, then selecting *Edit HelmChartRepository* through the kebab menu for your Helm chart repository, an *Error* page was displayed that showed a *404: Page Not Found* error. This was caused by a component path that was not up to date. This issue is now fixed. (link:https://issues.redhat.com/browse/OCPBUGS-14660[*OCPBUGS-14660*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:1693]:
```suggestion
* Previously, if you tried to edit a Helm chart repository in the *Developer* console by navigating to *Helm*, clicking the *Repositories* tab, then selecting *Edit HelmChartRepository* through the {kebab} menu for your Helm chart repository, an *Error* page was displayed a `404: Page Not Found` error. This was caused by a component path that was not up to date. This issue is now fixed. (link:https://issues.redhat.com/browse/OCPBUGS-14660[*OCPBUGS-14660*])
```

**Review by jeana-redhat — COMMENTED** [review]
nit on one of these, otherwise lookin' good :nerd_face: 

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: release_notes/ocp-4-14-release-notes.adoc (hunk 5 above)
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: fixed, thanks!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: fixed.

**Pull request merged**